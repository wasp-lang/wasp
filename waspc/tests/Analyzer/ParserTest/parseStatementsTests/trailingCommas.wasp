test Decl {
  list: [ 1, ],
}
