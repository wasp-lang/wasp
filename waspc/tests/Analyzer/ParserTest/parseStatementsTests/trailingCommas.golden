(AST
  (Decl@1:1-3:1 type=test name=Decl
    (Dict@1:11-3:1
      (DictEntry key=list
        (List@2:9-14
          (Integer@2:11 value=1)
        )
      )
    )
  )
)
